src/sources/socket/mod.rs

```diff
@@ -1220,11 +1307,35 @@ mod test {
     }
 
     ////////////// UNIX TEST LIBS //////////////
+
     #[cfg(unix)]
     async fn init_unix(sender: SourceSender, stream: bool, use_vector_namespace: bool) -> PathBuf {
-        let in_path = tempfile::tempdir().unwrap().into_path().join("unix_test");
+        init_unix_inner(sender, stream, use_vector_namespace, None).await
+    }
+
+    #[cfg(unix)]
+    async fn init_unix_with_config(
```

Comment:
What to replicate the `init_udp_with_config` and `init_udp_inner` methods for testing with custom configuration (in order to set the gelf deserializer in the config)